packages/design-system/src/components/MonthPicker/MonthPicker.tsx

```diff
@@ -165,7 +165,7 @@ export const MonthPicker = (props: MonthPickerProps) => {
         </Button>
       </div>
       <div className="ds-c-month-picker__months">
-        <ol className="ds-c-list--bare ds-c-month-picker__months-list">
+        <ol role="list" className="ds-c-list--bare ds-c-month-picker__months-list">
```

Comment:
This role is already defined by `ol`, I think this can be removed?

Reply:
Fun HTML bug for Safari - when you style a list to remove the list style Safari removes the list role as well. So the recommendation to support Safari is to include the role when removing list styles.

Reply:
In short... Safari's gonna Safari.

Reply:
It might actually be a better solution in our DS to tie unstyled list styles to the `role="list"` attribute to ensure a11y gets included, but maybe that's overkill? Gonna post in GH to discuss.

Reply:
oh wow, that's strange!